curriculum/challenges/english/25-front-end-development/lecture-understanding-performance-in-web-applications/67d2f778a856c37bd31c5022.md

```diff
@@ -2,13 +2,49 @@
 id: 67d2f778a856c37bd31c5022
 title: What Are Some Key Metrics for Measuring Performance?
 challengeType: 11
-videoId: nVAaxZ34khk
+videoId: 8hAnuI_pXqo
 dashedName: what-are-some-key-metrics-for-measuring-performance
 ---
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+What are some key metrics for measuring performace?
+
+To measure and improve web performance, we use key performance metrics like First Contentful Paint (or FCP), Speed Index, Total Blocking Time (or TBT), Bounce Rate, and Unique Users. Let's look at all of these metrics individually.
+
+First Contentful Paint, or FCP. This measures how quickly the first piece of content, like a text or image, appears on the screen. A good FCP is regarded as a time below 1.8 seconds, and a poor FCP is usually above 3 seconds. You can check your FCP using Chrome DevTools like this:
+
+Step 1: Open Chrome DevTools
+
+- Open Google Chrome.
+
+- Go to the webpage you want to test.
+
+- Press "F12" or Right-click and "Inspect".
+
+Step 2: Find and click the "Lighthouse" tab in the DevTools.
+
+Step 3: Click the "Analyze page load" button and wait for the analyzation to finish.
```

Comment:
Why step 2 and 3 are not in the video? I mean they are not described.

Reply:
I had to improvise a little here. I think it's fine for now, but we could make an issue about it.